helm/values-dev.yaml

```diff
@@ -76,7 +76,7 @@ microservice-chart:
     REDIS_PORT: "6380"
     REDIS_SSL_ENABLED: "true"
     NODO_HOSTNAME: https://api.dev.platform.pagopa.it
-    NODE_FOR_PSP_URI: /nodo/node-for-psp/v1
+    NODE_FOR_PSP_URI: /nodo-auth/node-for-psp/v1
```

Comment:
Missing `NODO_NODEFORPSP_API_KEY` and `NODO_CLOSEPAYMENT_API_KEY` env secrets?

Reply:
fixed